src/features/meetings/hooks/usePlaceSearch.ts

```
@@ -0,0 +1,163 @@
+    setSearchState('idle')
+    setHoveredPlaceId(null)
+    setHasBeenSearched(false)
+    setMapInstance(null)
+    reset()
+  }, [reset])
+
+  const handleKeyDown = useCallback(
+    (e: React.KeyboardEvent<HTMLInputElement>) => {
+      if (e.key !== 'Enter') return
+      // 한국어 등 IME 입력 중 composition 이벤트는 무시
+      if (e.nativeEvent.isComposing) return
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)